File: backend/src/main/java/com/lovepaws/app/api/dto/MascotaDTO.java
package com.lovepaws.app.api.dto;

public class MascotaDTO {
    private Long id;
    private String nombre;
    private Long razaId;
    private Long categoriaId;
    private Integer edad;
    private String sexo;
    private String descripcion;
    private String fotoUrl;
    private Long estadoId;

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public String getNombre() { return nombre; }
    public void setNombre(String nombre) { this.nombre = nombre; }
    public Long getRazaId() { return razaId; }
    public void setRazaId(Long razaId) { this.razaId = razaId; }
    public Long getCategoriaId() { return categoriaId; }
    public void setCategoriaId(Long categoriaId) { this.categoriaId = categoriaId; }
    public Integer getEdad() { return edad; }
    public void setEdad(Integer edad) { this.edad = edad; }
    public String getSexo() { return sexo; }
    public void setSexo(String sexo) { this.sexo = sexo; }
    public String getDescripcion() { return descripcion; }
    public void setDescripcion(String descripcion) { this.descripcion = descripcion; }
    public String getFotoUrl() { return fotoUrl; }
    public void setFotoUrl(String fotoUrl) { this.fotoUrl = fotoUrl; }
    public Long getEstadoId() { return estadoId; }
    public void setEstadoId(Long estadoId) { this.estadoId = estadoId; }
}

File: app/src/main/java/com/cibertec/applovepaws/feature_mascota/data/dto/MascotaDto.kt
package com.cibertec.applovepaws.feature_mascota.data.dto

class MascotaDto(
    val id: Int,
    val nombre: String,
    val edad: Int,
    val sexo: String,
    val descripcion: String?,
    val fotoUrl: String?,

    val categoriaId: Int?,
    val categoriaNombre: String?,

    val razaId: Int?,
    val razaNombre: String?,

    val estadoId: String?,
    val estadoDescripcion: String?,

    val usuarioCreacionId: Int?
)